### ISSUE TITLE
Verbose flag not working with `run`

### ISSUE THREAD

**Issue by papb** [python-poetry/poetry#3675]
- [X] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [X] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [X] <s>If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).</s> *Impossible to do precisely due to the issue*

- **OS version and name**: Windows 7
- **Poetry version**: 1.1.4
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: Not needed, full reproduction steps given

## Issue

1. Go to an empty directory and run `poetry init`, press `<Enter>` several times until it is done
2. `poetry run -vvv python`

See the error:

```
FileNotFoundError

  [WinError 2] The system cannot find the file specified

  at C:\Python39\lib\subprocess.py:1416 in _execute_child
      1412│             sys.audit("subprocess.Popen", executable, args, cwd, env)
      1413│
      1414│             # Start the process
      1415│             try:
    → 1416│                 hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
      1417│                                          # no special security
      1418│                                          None, None,
      1419│                                          int(not close_fds),
      1420│                                          creationflags,
```

**Comment by sinoroc** [comment]
I feel like it's a duplicate ticket. Or at least I am pretty sure I have seen (or even maybe participated in) a ticket discussion about inconsistencies of the CLI. Maybe that is covered by the new version of cleo (a dependency of poetry) that is being integrated these days.

**Comment by branchvincent** [comment]
Thanks for the report, this should be fixed in `master` and will be released in `1.2`. See https://github.com/python-poetry/poetry/issues/1567#issuecomment-1053210055
